caller/textractcaller/t_call.py

```diff
@@ -102,6 +102,31 @@ def get_dict(self):
         else:
             return {}
 
+@dataclass
+class Adapter():
+    adapter_id: str
+    version: int
```

Comment:
The version datatype schould match the API data type and the adapter version is a string in the documentation:
https://docs.aws.amazon.com/textract/latest/dg/API_GetAdapterVersion.html
```
{
   "AdapterId": "string",
   "AdapterVersion": "string"
}
```

better calling it adapter_version as well, which will be easier to map to the AdapterVersion mame

Reply:
Hey @schadem 
I modeled it based on the AnalyzeDocument API, which uses Version and not AdapterVersion. - https://docs.aws.amazon.com/textract/latest/dg/API_AnalyzeDocument.html
Let me know your thoughts